### PR TITLE
Wrap naked PQerrorMessage calls in libpagestore with pchomp

### PR DESCRIPTION
**Pull request by tristan957**
## Describe your changes

Remove the trailing newline prior to logging the message

## Issue ticket number and link

https://github.com/neondatabase/neon/pull/4239#issuecomment-1548383013

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] If it is a core feature, I have added thorough tests.
- [x] Do we need to implement analytics? if so did you add the relevant metrics to the dashboard?
- [x] If this PR requires public announcement, mark it with /release-notes label and add several sentences in this section.

## Checklist before merging

- [x] Do not forget to reformat commit message to not include the above checklist

### CODE REVIEW

**Comment by tristan957** [comment]
> Do not forget to reformat commit message to not include the above checklist

Does that mean I should remove only the checklists when finally committing? Or cut the whole thing down to just commit message and description. I might've messed up my first commit :).

File: pgxn/neon/libpagestore.c [hunk 1]
```diff
@@ -192,8 +192,9 @@ call_PQgetCopyData(char **buffer)
 		{
 			if (!PQconsumeInput(pageserver_conn))
 			{
-				neon_log(LOG, "could not get response from pageserver: %s",
-						 PQerrorMessage(pageserver_conn));
+				char	   *msg = pchomp(PQerrorMessage(pageserver_conn));
```
Comment (tristan957) [inline pgxn/neon/libpagestore.c:195]: Not entirely sure I have caught on to how whitespace should be handled. I presume this is following a Postgres style guide? I haven't read that yet.

Comment (hlinnaka) [reply pgxn/neon/libpagestore.c:195]: Yeah, we use the Postgres style in `pgxn/`. We haven't automated though, so there are many places that violate it in practice.

See https://www.postgresql.org/docs/current/source-format.html for the Postgres style guide. `pgindent`, in `src/tools/pgindent/`, can be used to format code to follow the Postgres style.

**Review by hlinnaka — APPROVED** [review]
LGTM

**Comment by hlinnaka** [comment]
> > Do not forget to reformat commit message to not include the above checklist
> 
> Does that mean I should remove only the checklists when finally committing? Or cut the whole thing down to just commit message and description. I might've messed up my first commit :).

Ah yeah, you should generally remove all the checklists. They're not very interesting to remember in the git history.

**Comment by tristan957** [comment]
Learning is a process :).

**Pull request merged**